### PR TITLE
Updated lr-align image to use minimap2_24

### PR DESCRIPTION
**Pull request by jonn-smith**
- Updated `minimap2` to version `2.24`
- Updated `lr-align` image to version `0.1.28`
- Updated all WDLs to use `lr-align:0.1.28`

### CODE REVIEW

File: docker/lr-align/Dockerfile [hunk 1]
```diff
@@ -23,10 +23,10 @@ COPY merge_read_group_tags.py /usr/local/bin/
 COPY cat_as_fastq.py /usr/local/bin/
 
 # install minimap2
-RUN wget https://github.com/lh3/minimap2/releases/download/v2.17/minimap2-2.17_x64-linux.tar.bz2 \
-        && tar xjf minimap2-2.17_x64-linux.tar.bz2 \
-        && rm minimap2-2.17_x64-linux.tar.bz2 \
-        && cd minimap2-2.17_x64-linux \
+RUN wget https://github.com/lh3/minimap2/releases/download/v2.24/minimap2-2.24_x64-linux.tar.bz2 \
```
Comment (SHuang-Broad) [inline docker/lr-align/Dockerfile:26]: minor: define an `ARG` for these version info? easier to maintain.

File: docker/lr-align/Makefile [hunk 2]
```diff
@@ -1,14 +1,18 @@
-VERSION = 0.1.27
+VERSION = 0.1.28
```
Comment (SHuang-Broad) [inline docker/lr-align/Makefile:1]: while you are here, let's take away the quay repo.

File: wdl/tasks/AlignReads.wdl [hunk 3]
```diff
@@ -104,16 +104,16 @@ task SAMtoPAF {
         filename=$(basename -- ~{sam_formatted_file})
         extension="${filename##*.}"
         if [[ "$extension" == "sam" ]]; then
-            /minimap2-2.17_x64-linux/k8 \
-                /minimap2-2.17_x64-linux/paftools.js \
+            /minimap2-2.24_x64-linux/k8 \
```
Comment (SHuang-Broad) [inline wdl/tasks/AlignReads.wdl:107]: let's use a variable/macro for that here as well.

**Review by SHuang-Broad — APPROVED** [review]
Some minor issues, good to go.

File: docker/lr-align/Makefile (hunk 2 above)
Comment (jonn-smith) [reply docker/lr-align/Makefile:1]: 👍

File: docker/lr-align/Dockerfile (hunk 1 above)
Comment (jonn-smith) [reply docker/lr-align/Dockerfile:26]: 👍

File: wdl/tasks/AlignReads.wdl (hunk 3 above)
Comment (jonn-smith) [reply wdl/tasks/AlignReads.wdl:107]: 👍

**Comment by SHuang-Broad** [comment]
@jonn-smith note that some of the WDLs don't validate.

**Pull request merged**